documentation/docs/configuration.md

```diff
@@ -241,6 +241,25 @@ if no match is loaded. Set to empty string to disable. **`Default: ""`**
 ####`get5_warmup_cfg`
 :   Config file executed in warmup periods. **`Default: get5/warmup.cfg`**
 
+####`get5_demo_upload_url`
+:   If defined, it is the URL at which a server resides to accept connections to upload demos. Requires
+[SteamWorks](../installation/#steamworks) extension. If no protocol is provided, the plugin will prepend
+`http://` to this value.
+
+####`get5_demo_upload_header_key`
+:   If defined, it is the authorization key that is appended to the HTTP request. Requires
+[SteamWorks](../installation/#steamworks) extension. **`Default: Authorization`**
+
+####`get5_demo_upload_header_value`
+:   If defined, it is the authorization value that is appended to the HTTP request. Requires
+[SteamWorks](../installation/#steamworks) extension. This is where your API *key* would
+be set.
+
+####`get5_demo_upload_delete_after_upload`
```

Comment:
Missing **`Default: ""`**

Reply:
0 means disabled. Much like all the other bool values they are default to 0.

Reply:
Ah yes of course this was a bool. I’m ok my phone and got them mixed up.